packages/mobile_app/lib/app_global.dart

```diff
@@ -69,8 +69,8 @@ class _Firebase {
 
 class _Ads {
   AdManagerInterstitialAd manager;
-  bool adsWatched = false;
-  bool adsLoaded = false;
+  bool adsAreWatched = false;
+  bool adsAreLoaded = false;
```

Comment:
`AreWatched` — this form still does not correspond to what it does. correctly it would haveBeenWatched but it is very long. so I simplified